src/main/java/org/folio/circulation/domain/policy/OverdueFinePolicy.java

```diff
@@ -38,7 +69,40 @@ public static OverdueFinePolicy unknown(String id) {
 
   private static class UnknownOverdueFinePolicy extends OverdueFinePolicy {
     UnknownOverdueFinePolicy(String id) {
-      super(id, null, null, null);
+      super(id, null, null, null, null, null, null, null);
+    }
+  }
+
+  public enum OverdueFineInterval {
```

Comment:
Can the enum be extracted to a separate file?

Reply:
Done